ios_widget_catalog_compare/lib/main.dart

```diff
@@ -59,14 +38,268 @@ class _FlutterDemoState extends State<FlutterDemo> {
     _platformEventChannel.receiveBroadcastStream().listen((dynamic name) {
       if (name != controlName) setState(() => controlName = name);
     });
+
+    textController = TextEditingController(text: '');
   }
 
   @override
   Widget build(BuildContext context) {
     return CupertinoPageScaffold(
       child: Center(
-        child: widgetMap[controlName],
+        child: Padding(
+          padding: EdgeInsetsDirectional.fromSTEB(8.0, 8.0, 8.0, 8.0),
+          child: widgetPicker(context),
+        ),
       ),
     );
   }
+
+  Widget widgetPicker(BuildContext context) {
+    switch (controlName) {
+      case 'CupertinoButton':
+        return CupertinoButton(
+          child: Text('Button'),
+          onPressed: () {/** */},
+        );
+
+      case 'CupertinoTextField':
+        return CupertinoTextField(
+          placeholder: "Placeholder",
+        );
+
+      case 'CupertinoPicker':
+        return CupertinoPicker(
+          scrollController: FixedExtentScrollController(initialItem: 3),
+          magnification: 1.2,
+          useMagnifier: true,
+          itemExtent: 32.0,
+          onSelectedItemChanged: (value) {},
+          children: const [
+            Text('One'),
+            Text('Two'),
+            Text('Three'),
+            Text('Four'),
+            Text('Five'),
+          ],
+        );
+
+      case 'CupertinoSearchTextField':
+        return CupertinoSearchTextField(
+          controller: textController,
+          onChanged: (value) {
+            print("The text has changed to: " + value);
+          },
+          onSubmitted: (value) {
+            print("Submitted text: " + value);
+          },
+        );
+
+      case 'CupertinoFormSection.groupInset':
+        return Form(
```

Comment:
Maybe put this in a separate widget class, like a CupertinoFormSectionGroupInsetDemo.